assets/js/modules/analytics/components/settings/GA4SettingsControls.js

```diff
@@ -88,12 +88,7 @@ export default function GA4SettingsControls( {
 		select( CORE_SITE ).getDocumentationLinkURL( 'ga4' )
 	);
 
-	const isGA4Connected = useSelect( ( select ) =>
-		select( CORE_MODULES ).isModuleConnected( 'analytics-4' )
-	);
-
-	const hasModuleAccess =
-		isGA4Connected && hasAnalyticsAccess && hasAnalytics4Access;
+	const hasModuleAccess = hasAnalyticsAccess && hasAnalytics4Access;
```

Comment:
I find this a little ambiguous to understand in its use but I suppose it's ok for now so long as it's working properly as it seems to be :)